test/t_case_gb.cc

```diff
@@ -11,6 +11,8 @@
 
 namespace ilang {
 
+#ifdef ILANG_BUILD_SYNTH
```

Comment:
Use the ILA portable for case-specific tests. The consistency check between abstraction and portable should be checked in the interface tests.